frontend/test/metabase/scenarios/permissions/reproductions/19603-archived-sub-collection-shows-up-in-permissions.cy.spec.js

```diff
@@ -0,0 +1,22 @@
+import { restore } from "__support__/e2e/cypress";
+
+describe.skip("issue 19603", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    // Archive second collection (nested under the first one)
+    cy.request("GET", "/api/collection/").then(({ body }) => {
+      const { id } = body.find(c => c.slug === "second_collection");
```

Comment:
I've added a helper for that in my PR

Reply:
I'd like us to have a bunch of helpers so that we can obtain id of a collection by its slug, name, etc.